docs/learn/betting_strategy.md

```diff
@@ -0,0 +1,269 @@
+buying the same amount of all other outcome tokens. This means you're betting
+that $A$ is less likely to occur than currently predicted by the market. In
+particular, it is absolutely a valid strategy to own multiple outcome tokens in
+a market.
+
+The farther off the price is from your prediction, the higher the profit. Note
+that, as discussed above, _we're assuming that your prediction is correct_. The
+more your prediction diverges from the actual probabilities, the higher the risk
+of making a loss.
+
+## Trading Scalar Markets
+
+Suppose there's a scalar market with a range of $[a, b]$. Let $p$ and $q$ be the
+spot prices of SHORT and LONG, resp. Recall that $p + q = 1$. The value
+predicted by the market is $v = pa + qb$. Assume you believe that the market
+will resolve to $w \in [a, b]$. Given the current status $p, q, v$ of the
+market, how should you bet?
+
+The SHORT token takes on more value as the value that the market resolves to is
+closer to the lower bound $a$, and LONG tokens on more value as the value that
+the market resolves to is closer to the upper bound $b$. In fact, suppose that
+your belief is correct and the market resolves to $w$. Then each LONG is worth
+$(w - a) / (b - a)$ and each SHORT is worth $(b - w) / (b - a)$.
+
+Thus, if the market resolves to $w$, then the profit of buying one unit of LONG
+at the price of $q$ and later redeeming it is
+
+$$
+    \frac{w - a}{b - a} - q = \frac{w - v}{b - a}
+$$
+
+(using $v = pa + qb = a + q(b - a)$; a negative profit means a loss, of course).
+Thus, the trade is good whenever $w > v$. In other words, a reasonable informant
+buys LONG if they believe the prediction is too low. Note that the result of
+this buy is that the price of LONG increases, as does the value predicted by the
+scalar market, reflecting that the informant has provided information to the
+market. Since buying LONG is equivalent to selling/shorting SHORT, if you
+already own SHORT tokens, you can sell SHORT tokens in this situation instead of
+buying more LONG tokens.
+
+Similarly, the profit of buying one unit of SHORT at the price of $p$ is
+$(v - w) / (b - a)$, so a reasonable informant buys SHORT if they believe the
+prediction is too high.
+
+If $w$ lies outside of the scalar range $[a, b]$, then just assume that $w = a$
+if $w < a$ or $w = b$ if $w > b$ and apply the system above.
+
+## Providing Liquidity
+
+Trades on prediction markets shift the price of the assets. Buying (resp.
+selling) an outcome asset increases (resp. decreases) its price and decreases
+(resp. increases) the prices of the other outcome assets. By buying (resp.
+selling) you bet that the probability of the outcome is higher (resp. lower)
+than currently predicted by the market. But how can you expose yourself to the
+market if you think that the prices are correct? This is where providing
+liquidity comes into play.
+
+The liquidity of a market is important when it comes to creating accurate
```

Comment:
```suggestion
The liquidity of a market is important for creating accurate
```

Reply:
What's the problem here?

Reply:
just a better language suggested by chatgpt

Reply:
7d331db36244dfa97b4c3b6ef6ced64f94700951

And that's the only thing the fella found objectionable...? 🍉

Reply:
Seems like you are a genius.